src/pages/projectSettings/CopyProjectModal.tsx

```diff
@@ -58,11 +58,13 @@ export const CopyProjectModal: React.VFC<Props> = ({
           { shouldTimeout: false }
         );
       } else {
-        success(`Successfully created the project: ${identifier}`);
+        success(`Successfully duplicated the project: ${identifier}`);
```

Comment:
I know this was here before but can you refactor this to use the `dispatchToast.status()` pattern.
Just so we have consistent usage through out the code base.